packages/core/content-manager/server/src/preview/controllers/index.ts

```diff
@@ -0,0 +1,6 @@
+import type { Plugin } from '@strapi/types';
+import { createPreviewController } from './preview';
+
+export const controllers = {
+  preview: createPreviewController,
+} as unknown as Plugin.LoadedPlugin['controllers'];
```

Comment:
Why is unknown needed?

and I thought you would need to call the function, I didn't know this style was allowed

Reply:
apparently the controller types do not account for using the factory itself, but it works (copied it from content history)

I will add a comment!